backends/memcache.go

```diff
@@ -173,3 +173,14 @@ func (m *Memcache) SRem(key string, value string) (bool, error) {
 	return false, errors.New("mantle: SETS/SRem unimplemented for memcache datastore")
 }
 
+// Raise unimplemented error
+func (m *Memcache) Sismember(key string, member string) (bool, error) {
+	return false, errors.New("mantle: SETS/Sismember unimplemented for memcache datastore")
+}
+
+func (m *Memcache) Sismembers(key string, members []string) ([]bool, error) {
```

Comment:
For 2. Instead of returning this "errors.New("mantle: SETS/Sismember unimplemented for memcache datastore")", we can do a get(key) and check if it exists. NotImpleted error is not so useful IMO.